plone/app/locales/locales/it/LC_MESSAGES/widgets.po

```diff
@@ -465,15 +465,15 @@ msgstr "Riordina per"
 
 #: pat/preventdoublesubmit/preventdoublesubmit.js
 msgid "You already clicked the submit button. Do you really want to submit this form again?"
-msgstr ""
+msgstr "Hai già cliccato il bottone per l'invio. Vuoi realmente inviare ancora questa form?"
```

Comment:
```suggestion
msgstr "Hai già cliccato il bottone per l'invio. Vuoi realmente inviare ancora questo form?"
```

Pare che sia preferito il maschile che e' anche consistente con le altre traduzioni.
https://accademiadellacrusca.it/it/consulenza/genere-dei-forestierismi/1229#:~:text=Attualmente%20form%20%C3%A8%20presente%20solo,GARZANTI%20che%2C%20pur%20indicando%20come

In altri posti viene utilizzato `Modulo`.

Reply:
Done, also with modulo instead of form